Userland/Libraries/LibC/stdio.cpp

```diff
@@ -788,6 +792,8 @@ int fputc(int ch, FILE* stream)
     return byte;
 }
 
+alias(__fputc, fputc);
```

Comment:
Doing this to only some of the functions is weird, and doing it to all of them sounds like a waste...
Is there no other way to disable the nonnull attribute here? e.g. by manually giving it some attributes, maybe?

Reply:
The documentation seems to indicate that it's impossible if nonnull is applied, which GCC does automatically.

My only other idea is to compile this specific file with `-fno-builtin=fread -fno-builtiin=fputc -fno-builtin=fputs`

Reply:
IIRC there is no such pragma, only `diagnostic`, `target`, `optimize` and `unroll` can be configured via the preprocessor.